content/pl/build-release-run.md

```diff
@@ -0,0 +1,18 @@
+## V. Buduj, wypuszczaj, uruchamiaj
+### Precyzyjnie oddziel etap budowania od uruchamiania
+
+[Codebase](./codebase) jest przetwarzany w (nie developerskie) wdrożenie w trzech etapach
```

Comment:
Gdyby `developerskie` było polskim przymiotnikiem, to powiedziałbym że powinno być `niedeveloperskie`. Ale tutaj nie wiem szczerze mówąc.